modules/visual_script/doc_classes/VisualScriptBuiltinFunc.xml

```diff
@@ -204,7 +204,10 @@
 		<constant name="COLORN" value="62" enum="BuiltinFunc">
 			Return the [Color] with the given name and alpha ranging from 0 to 1. Note: names are defined in color_names.inc.
 		</constant>
-		<constant name="FUNC_MAX" value="63" enum="BuiltinFunc">
+		<constant name="MATH_SMOOTHSTEP" value="63" enum="BuiltinFunc">
+			Hermite interpolation.
```

Comment:
Would be good to add more details about what this means.

Reply:
Fixed to "Return a number smoothly interpolated between the first two inputs, based on the third input. Using Hermite interpolation formula : float t; t = clamp((weight - from) / (to - from), 0.0, 1.0); return t * t * (3.0 - 2.0 * t);"